Backend/src/test/java/com/senla/repository/EventRepositoryTest.java

```diff
@@ -0,0 +1,27 @@
+package com.senla.repository;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.senla.model.Event;
+import com.senla.model.User;
+import java.util.Optional;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+
+class EventRepositoryTest extends DatabaseTest {
+
+    @Autowired private UserRepository userRepository;
+
+    @Autowired private EventRepository eventRepository;
+
+    @Test
+    void findByAuthorId() {
+        Optional<User> user = userRepository.findByEmail("user@gmail.com");
+        Event event =
+                Event.builder().name("Hello").description("Hello world").author(user.get()).build();
+        eventRepository.save(event);
+        Page<Event> foundEvent = eventRepository.findByAuthorId(user.get().getId(), null);
+        assertThat(foundEvent.get().findFirst().get().getName()).isEqualTo("Hello");
```

Comment:
I suggest to use junit5 Assertions:

Assertions.assertEquals(foundEvent.get().findFirst().get().getName(), "Hello");

```xml
<dependency>
    <groupId>org.junit.jupiter</groupId>
    <artifactId>junit-jupiter-api</artifactId>
    <version>5.9.0</version>
    <scope>test</scope>
</dependency>
```